### ISSUE TITLE
Transition not happening when close

### ISSUE THREAD

**Issue by vdiaza** [fat/zoom.js#95]
When I click on the image the zoom transition works great and smooth, but when I close it or scroll the image just go back to normal without any transition.

You know what could it be causing this?
the console does not show errors at all.

**Comment by rickydonald** [comment]
Yes, I had the same issue. I tried changing the bootstrap bundle js version from 5.2.2 to 3.0.0 and it worked for me. I still need to figure out what's missing on 5.2.2 bundle.js
